delta/plugins/elasticsearch/src/main/resources/defaults/default-settings.json

```diff
@@ -5,26 +5,13 @@
         "type": "custom",
         "tokenizer": "classic",
         "filter": [
-          "my_multiplexer"
+          "trim",
+          "word_delimiter_nexus",
+          "lowercase"
         ]
       }
     },
     "filter": {
-      "my_multiplexer": {
-        "type": "multiplexer",
-        "filters": [
-          "lowercase",
-          "uuid_type_4",
-          "word_delimiter_nexus"
-        ]
-      },
-      "uuid_type_4": {
```

Comment:
we don't need this anymore?

Reply:
we added that to be able to search resources by uuids in fusion admin

Reply:
No, `word_delimiter_nexus` is splitting as expected for UUIDs already.